cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -806,32 +813,34 @@ def send_data(tls, kube_api_endpoint):
     if extra_sans and not extra_sans == "":
         sans.extend(extra_sans.split())
 
-    # Create a path safe name by removing path characters from the unit name.
-    certificate_name = hookenv.local_unit().replace('/', '_')
     # Request a server cert with this information.
-    tls.request_server_cert(common_name, sans, certificate_name)
+    tls_client.request_server_cert(common_name, sans,
+                                   crt_path=server_crt_path,
+                                   key_path=server_key_path)
+
+    # Request a client cert for kubelet.
+    tls_client.request_server_cert('system:kube-apiserver',
```

Comment:
client cert/server cert... confusing naming here.

Reply:
Supposed to be `tls_client.request_client_cert` I think.

Reply:
It absolutely is.  My mistake.